server/index.js

```diff
@@ -14,7 +14,7 @@ keystone.init({
   'auto update': true,
   'session': true,
   'auth': true,
-  'mongo': process.env.MONGODB_URL || 'mongodb://127.0.0.1/mmdp-cms',
+  'mongo': 'mongodb://localhost/keystonereactcms',
```

Comment:
I see the initial value of the mongo property has been changed. The environment variable is quite important for production when the username and password of the database will be included as part of the database connection URI. For example: `mongodb://<dbuser>:<dbpassword>@ds259499.mlab.com:59499/employees`.

Also, I feel we shouldn't use the names of the technologies used as our database name as it isn't descriptive of what the database is used for.
```suggestion
  'mongo': 'mongodb://localhost/<another-name>',
```
Kindly help look into the database name and adding of support for the database URL environment variable. Well-done.

Reply:
Thank you for the feedback. I'll refactor that code.

Reply:
Alright. You are welcome.

Reply:
I have refactored the code.

Reply:
So as we do not create a precedence of assumptions, the db path and the cookie secret should strictly be in the .env and picked as env variables

Reply:
Thank you for the request. I have added a .env_sample, that will guide the team on environment variables needed for the app to run normally.